torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -1031,6 +1030,279 @@ std::pair<TensorDomain*, TensorDomain*> TensorDomain::rFactor(
       TransformRFactor::runReplay2(this, axes)};
 }
 
+namespace {
+
+/** \brief Container class DisjointSet models equivalence relationships
```

Comment:
@tlemo Is this the right style for Doxygen?

Reply:
First, nice job adding Doxygen comments in the first place!

@shmsong, I sent the Doxygen guide link on team chat. A few quick notes:

1. Please only use // for comments, and //! for doxygen comments
2. The recommended style is to avoid explicit \brief and \details
 (rather place the brief description on the first line, then the detailed description following with an empty //! line in between. Example: https://github.com/csarofeen/pytorch/blob/7d72e8aba743df78682002671d73c4433a2f429e/torch/csrc/jit/codegen/cuda/ir_printer.h#L14-L27

Reply:
Thanks for the suggestions. Re-formatted the doxygen comments.